Tasks/AzureWebPowerShellDeployment/task.json

```diff
@@ -31,7 +31,7 @@
     {
       "name": "WebSiteName",
       "type": "pickList",
-      "label": "Web App Name",
+      "label": "Web App Name with out slot.If your web app uses slot, mention in **Slot** field, else default **Production** slot is considered",
```

Comment:
Grammar/verbiage:
```
Web App Name without a slot. If your web app uses a slot, mention in the **Slot** field, otherwise the default **Production** slot is used.
```